.doc_gen/metadata/dynamodb_metadata.yaml

```diff
@@ -1423,6 +1423,18 @@ dynamodb_Scenario_PartiQLSingle:
   synopsis: query a &DDB; table using PartiQL.

```

Comment:
Part of the scenario is to show also the single-api action examples for BatchExecuteStatement and Execute statement (see [Python example here](https://github.com/awsdocs/aws-doc-sdk-examples/tree/main/python/example_code/dynamodb#code-examples))

Reply:
This hasn't been implemented

Reply:
This is the piece that's not done